### PR TITLE
Make app/addon readmes consistent

### PR DESCRIPTION
**Pull request by elwayman02**
- Build "change directory" instruction dynamically from project name
- Update addon readme to match app readme language
- Update ember-try command from `testall` to `each`

/cc @rwjblue

### CODE REVIEW

File: blueprints/addon/files/README.md [hunk 1]
```diff
@@ -4,18 +4,19 @@ This README outlines the details of collaborating on this Ember addon.
...
+* `git clone <repository-url>` this repository
+* `cd <%= addonName %>`
 * `npm install`
 * `bower install`
 
 ## Running
 
 * `ember serve`
-* Visit your app at http://localhost:4200.
+* Visit your app at [http://localhost:4200](http://localhost:4200).
```
Comment (Turbo87) [inline blueprints/addon/files/README.md]: can we use `<http://localhost:4200>` here instead?

Comment (elwayman02) [inline blueprints/addon/files/README.md]: I don't have a strong preference, I just copied how it's written in the [app README](https://github.com/ember-cli/ember-cli/blame/master/blueprints/app/files/README.md#L26)

Comment (Turbo87) [inline blueprints/addon/files/README.md]: ah okay, nevermind then

**Comment by nathanhammond** [comment]
@homu r+

**Pull request merged**